packages/ui/core-components/src/lib/unsorted/ui/Link.svelte

```diff
@@ -0,0 +1,20 @@
+<script context="module">
+	export const evidenceInclude = true;
+</script>
+
+<script>
+	export let url = '#'; // Destination URL
+	export let label = 'Click here'; // Link text
+	export let newTab = false; // Open in a new tab
+	$: newTab = newTab === 'true' || newTab === true;
```

Comment:
@zachstence has created a nice isBoolean function in utils that might be helpful here